### ISSUE TITLE
Private messages don't work

### ISSUE THREAD

**Issue by mayfield** [SauceLLC/sauce4zwift#6]
We're getting the data stream from a different account so they simply don't come down the wire.

Will have to fallback to companion protocol and/or polling some API that gives us access to this.

**Comment by ivodam2** [comment]
Yes generally the same problem, but Zwift Companion doesn`t go into Game mode at all when Sauce is active. At least in **banded** rides. When Sauce is off - Companion Game mode works. I am not the only one. Got feedback from several riders. I am on Android. Don`t know about Apple.

**Comment by mayfield** [comment]
> Yes generally the same problem, but Zwift Companion doesn`t go into Game mode at all when Sauce is active. At least in **banded** rides. When Sauce is off - Companion Game mode works. I am not the only one. Got feedback from several riders. I am on Android. Don`t know about Apple. But....... on Sauce message board messages from Companion Game mode are coming, they are visible on screen, only we can not type anything since Game mode on Android does not work.

This is unfortunately how the companion protocol works with Zwift.  Only one connection at a time is allowed.  This is why it's off by default.
